impl/LogicalPathApiImpl.cpp

```diff
@@ -0,0 +1,52 @@
+/**
+* iRODS REST API
+* This is the iRODS REST API
+*
+* OpenAPI spec version: 1.0.0
+* Contact: info@irods.org
+*
+* NOTE: This class is auto generated by the swagger code generator program.
+* https://github.com/swagger-api/swagger-codegen.git
+* Do not edit the class manually.
+*/
+
+#include "LogicalPathApiImpl.h"
+#include <string>
+
+#include "utils.hpp"
+
+namespace io::swagger::server::api
+{
+    using namespace io::swagger::server::model;
+
+    LogicalPathApiImpl::LogicalPathApiImpl(Pistache::Address addr)
+        : LogicalPathApi(addr)
+        , irods_logical_path_{}
+    {
+    }
+
+    void LogicalPathApiImpl::delete_handler_impl(const Pistache::Rest::Request& request,
+                                    Pistache::Http::ResponseWriter& response )
+    {
+        auto irods_logic = [&]
```

Comment:
Try to capture only the things that are needed. Here, I think you only need to capture the `this` pointer.
```cpp
const auto irods_logic = [this]( // Rest of the lambda ...
```
Same applies to the other lambda in `rename_handler_impl`.